pyanaconda/core/event_loop.py

```diff
@@ -0,0 +1,58 @@
+# Event loop object.
+#
+# Use these only if there is no other way.
```

Comment:
This comment is a little confusing. Shouldn't we encourage people to use the core modules? Same for other files.

Reply:
Yes it shouldn't be here. It is leftover. I want this text only in `core/glib.py` to force developers to use abstraction rather than using glib functions directly.

Reply:
Ok.